eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -157,10 +157,7 @@ stages:
                         parameters:
                           paths:
                             - .github/CODEOWNERS
-                      - template: /eng/common/pipelines/templates/steps/get-pr-owners.yml
-                        parameters:
-                          TargetVariable: "OwningGHUser"
-                          ServiceDirectory: ${{parameters.ServiceDirectory}}
+
                       - template: /eng/common/pipelines/templates/steps/docs-metadata-release.yml
```

Comment:
don't we need to use the new template?

Reply:
Not yet. That change will be needed later when we do daily doc previews from the packages released to the dev feed.

The `docs-metadata-release.yml` no longer opens PRs, it pushes directly to the target branch. 

This cleanup work was done to remove extra steps that require network traffic but have no effect because the outputs (in this case github CODEOWNERS for a review) are not used.